Source/Xamarin/Prism.Forms/Navigation/IMultiPageNavigationAwareAsync.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
I don't want to add any async capabilities to this feature yet.  We should remove this until it is absolutely needed.  I am concerned about impacting tab changing while running a developers code and waiting for the task to return